app/components/views/Home/ThemeToggle.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from "react";
+import { ThemeToggle } from "@klimadao/lib/components";
+
+// To prevent loading errors need to wrap external ESM again
+// as it is loaded dynamically later in Navigation
+// https://github.com/vercel/next.js/issues/25454#issuecomment-862571514
+
+interface Props {
+  className?: string;
+}
+
+const ThemeToggleExport: FC<Props> = (props) => (
+  <ThemeToggle className={props.className} />
+);
```

Comment:
What are your thoughts on moving `<ThemeToggleExport />` to the shared lib directory?

Reply:
Theme toggle is in shared lib directory but imported and re-exported in site bc of this bug ```// To prevent loading errors need to wrap external ESM again
// as it is loaded dynamically later in Navigation
// https://github.com/vercel/next.js/issues/25454#issuecomment-862571514```

Reply:
so I did the same thing for app but if there is a better way then I'm happy to implement it.

Reply:
But if its reexported isn't it the same problem as the first one? Namely that you cant reference the path in the dynamic import? `@klimadao/lib/components`

If you do need to re-export it, perhaps this saves on some boilerplate, rather than defining a new component?
`export { ThemeToggle } from "@klimadao/lib/components"`

Reply:
wait actually @0xAeterno are you sure we need to wrap in dynamic import at all? can't we just do a good old check for `typeof document === 'undefined'` and fallback to light mode? then we can ditch the reexports altogether